examples/utils/example-utils/src/migrationTool/sameContainerMigrationTool.ts

```diff
@@ -32,6 +40,32 @@ export class SameContainerMigrationTool extends DataObject implements ISameConta
 		return this._setContainerRef;
 	}
 
+	private _setLoadPausedContainerFn:
```

Comment:
I don't think the MigrationTool needs to participate in loading the paused container, and instead the Migrator can retain/encapsulate loading priviliges.  Which would let you avoid needing to pass callbacks/promises here.

E.g. when the Migrator witnesses the MigrationTool get "readyForMigration", it can use its modelLoader to spawn the client (basically where it calls `migratable.exportData()` today, it should instead spawn a new migratable with proper paused state and call exportData on that.

Technically could do it sooner (as soon as the PactMap acceptance completes and before the Quorum proposing) but it'll be nice to keep the flow linear for the first iteration.